.github/workflows/main.yml

```diff
@@ -14,6 +14,9 @@ jobs:
         submodules: true
     - name: Install Rust
       run: rustup update ${{ matrix.rust }} && rustup default ${{ matrix.rust }}
+    - uses: cargo-bins/cargo-binstall@main
+    - name: Install wasm-tools (need json-from-wast subcommand)
+      run: cargo binstall wasm-tools -y
```

Comment:
This can be quite a slow build to run on CI. It can be faster to just use the release directly.

See eg https://github.com/bytecodealliance/wasmtime/blob/main/.github/workflows/main.yml#L801.

Reply:
I'm using `cargo binstall` here which uses the pre-compiled binary from GitHub releases

Reply:
Ahh I missed that, nice.